lib/elixir_analyzer/test_suite/rpg_character_sheet.ex

```diff
@@ -0,0 +1,89 @@
+defmodule ElixirAnalyzer.TestSuite.RpgCharacterSheet do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise rpg-character-sheet
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+  alias ElixirAnalyzer.Constants
+
+  feature "welcome ends with IO.puts" do
+    type :essential
+    find :all
+    depth 1
+    comment Constants.rpg_character_sheet_welcome_ends_with_IO_puts()
+
+    form do
+      def welcome() do
+        _block_ends_with do
+          IO.puts(_ignore)
+        end
+      end
+    end
+  end
+
+  assert_call "run uses welcome" do
+    type :essential
+    called_fn name: :welcome
+    calling_fn module: RPG.CharacterSheet, name: :run
+    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
+  end
+
+  assert_call "run uses ask_name" do
+    type :essential
+    called_fn name: :ask_name
+    calling_fn module: RPG.CharacterSheet, name: :run
+    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
+  end
+
+  assert_call "run uses ask_class" do
+    type :essential
+    called_fn name: :ask_class
+    calling_fn module: RPG.CharacterSheet, name: :run
+    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
+  end
+
+  assert_call "run uses ask_level" do
+    type :essential
+    called_fn name: :ask_level
+    calling_fn module: RPG.CharacterSheet, name: :run
+    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
+  end
```

Comment:
> Maybe the two IO.inspect warrant that, because learning how that function works is super useful, but the other two feel more like :actionable. What do you think?

I agree with your reasoning and conclusions. That would mean this change, right?

```suggestion
  feature "welcome ends with IO.puts" do
    type :actionable
    find :all
    depth 1
    comment Constants.rpg_character_sheet_welcome_ends_with_IO_puts()

    form do
      def welcome() do
        _block_ends_with do
          IO.puts(_ignore)
        end
      end
    end
  end

  assert_call "run uses welcome" do
    type :actionable
    called_fn name: :welcome
    calling_fn module: RPG.CharacterSheet, name: :run
    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
  end

  assert_call "run uses ask_name" do
    type :actionable
    called_fn name: :ask_name
    calling_fn module: RPG.CharacterSheet, name: :run
    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
  end

  assert_call "run uses ask_class" do
    type :actionable
    called_fn name: :ask_class
    calling_fn module: RPG.CharacterSheet, name: :run
    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
  end

  assert_call "run uses ask_level" do
    type :actionable
    called_fn name: :ask_level
    calling_fn module: RPG.CharacterSheet, name: :run
    comment ElixirAnalyzer.Constants.rpg_character_sheet_run_uses_other_functions()
  end
```